client/python/cryoet_data_portal/tests/test_codegen.py

```diff
@@ -0,0 +1,24 @@
+from graphql import GraphQLObjectType, GraphQLSchema
+
+from cryoet_data_portal._codegen import SCHEMA_PATH, load_schema, parse_fields
+
+
+def test_load_schema():
```

Comment:
I still need much more test coverage for code generator. I think just a few more tests for each of the public functions might be good enough.